marklogic-data-hub/src/test/resources/scaffolding-test/myTestIngestStep.step.json

```diff
@@ -3,7 +3,11 @@
   "type": "INGEST",
   "version": 1,
   "options": {
-    "key": "val",
-    "key2": "val2"
-  }
+    "collections": ["myTestIngestStep"],
+    "outputFormat": "json"
+  },
+  "customHook" : { },
+  "language" : "zxx",
+  "modulePath" : "/path/to/your/step/module/main.sjs",
```

Comment:
maybe we can add subfolder "ingest for Ingest step"
 -> "modulePath" : "/path/to/your/step/ingest/module/main.sjs" 
and "mapping for Mapping step"
-> "modulePath" : "/path/to/your/step/mapping/module/main.sjs"